test/api/handlers_test.go

```diff
@@ -78,6 +78,10 @@ func TestHandlers(t *testing.T) {
 		{name: "Challenge-Response/Registration", test: testRegistration},
 		// GET /register, POST /register, GET /login, POST /login
 		{name: "Challenge-Response/Login", test: testLogin},
+		// GET /user/apikey, POST /user/apikey, DELETE /user/apikey/:apiKey
```

Comment:
I think it should be `/user/apikeys` seeing a user can have multiple?

Reply:
Sure, we're using a plural in uploads and downloads, so we can use one here as well.